models/RoutesParser.cfc

```diff
@@ -592,6 +612,27 @@ component accessors="true" threadsafe singleton {
 		}
 	}
 
+	/**
+	* return true if the metadata represents a function marked "noCbSwagger"
+	*/
+	private boolean function isAnnotatedWithTruthyNoCbSwaggerAttribute( required struct functionMetadata ) {
+		if ( !structKeyExists( functionMetadata, "noCbSwagger" ) ) {
```

Comment:
Let's call this function metadata `cbSwagger`, check for its existence, and then look for a `boolean` `false`.  This can open up additional inclusions in the future by using `cbSwagger` or `cbSwagger=true` on a function or component.  When we implement this, we could then do discovery of an entire site without includes/excludes by simply using the component/function MD

Reply:
So your function annotation would become:

```
function myFunction() cbSwagger=false{...}
```
or
```
/**
* @cbswagger false
**/
function myFunction(){...}
```

Reply:
addressed